tests/test_middleware.py

```diff
@@ -113,3 +113,28 @@ async def lifespan_only_app(scope: Scope, receive: Receive, send: Send) -> None:
     app = MessagePackMiddleware(lifespan_only_app)
     scope = {"type": "lifespan"}
     await app(scope, mock_receive, mock_send)
+
+
+@pytest.mark.asyncio
+async def test_packb_unpackb() -> None:
+    async def app(scope: Scope, receive: Receive, send: Send) -> None:
+        request = Request(scope, receive)
+        response = JSONResponse({"message": "Hello, World!"})
+
+        await response(scope, receive, send)
+        assert "unpacked" == await request.json()
```

Comment:
Just rejigging this a bit to fit with my other comment…

```suggestion
        request = Request(scope, receive)
        assert await request.json() == {"message": "unpacked"}

        response = JSONResponse({"message": "Hello, World!"})
        await response(scope, receive, send)
```